tests/integration/targets/aci_l3out_bgp_peer/tasks/main.yml

```diff
@@ -255,7 +258,7 @@
   - name: verify remote AS object has been created correctly
     assert:
       that:
-        - add_eth_bgp_peer.current.0.bgpPeerP.children.1.bgpAsP.attributes.asn == "65456"
+        - add_eth_bgp_peer.current.0.bgpPeerP.children.2.bgpAsP.attributes.asn == "65456"
     when: version.current.0.topSystem.attributes.version is version('4', '>=')
```

Comment:
Do we need the version check?

Reply:
As per the existing tasks, we need the version check.